src/communication/request_sync_client_server.hpp

```diff
@@ -116,6 +116,8 @@ namespace cubcomm
       sequenced_payload &operator= (sequenced_payload &&other);
       sequenced_payload &operator= (const sequenced_payload &) = delete;
 
+      sequenced_payload &operator= (std::nullptr_t);
```

Comment:
Oh.. I didn't know that we can make use of `std::nullptr_t` to reset a variable in this way. 

The following question is just to learn it. Thank you in advance:
- The way I see it, it could be confusing in that std::nullptr_t is assigned to a non-pointer variable. (seemingly assign a pointer value to a non-pointer variable?). It would be ok if it's used widely. Is it used widely?

Reply:
I drew inspiration from `unique_pts`'s:
```
unique_ptr& operator=(std::nullptr_t) ) noexcept;
```
I do agree it is a bit forced but it shows the intention to actually "obliterate" / "invalidate" an object instance. Akin to a "reset" function.